test/functional/mempool_sigoplimit.py

```diff
@@ -133,6 +136,45 @@ def test_sigops_limit(self, bytes_per_sigop, num_sigops):
         assert_equal(entry_parent['descendantcount'], 2)
         assert_equal(entry_parent['descendantsize'], parent_tx.get_vsize() + sigop_equivalent_vsize)
 
+    def test_sigops_package(self):
+        self.log.info("Test a overly-large sigops-vbyte hits package limits")
```

Comment:
nit eb8f58f5e4a249d55338304099e8238896d2316d

```suggestion
       self.log.info("Test a package whose sigop-adjusted vsize hits ancestor size limits")
```